pkg/api/serialization_test.go

```diff
@@ -458,14 +480,14 @@ func TestSpecificKind(t *testing.T) {
 	}
 	seed := int64(2703387474910584091) //rand.Int63()
 	for i := 0; i < fuzzIters; i++ {
-		t.Logf(`About to test %v with "v1"`, kind)
+		//t.Logf(`About to test %v with "v1"`, kind)
 		fuzzInternalObject(t, v1.SchemeGroupVersion, item, seed)
 		roundTrip(t, kapi.Codecs.LegacyCodec(v1.SchemeGroupVersion), item)
 	}
 }
 
 // Keep this in sync with the respective upstream set
-var nonInternalRoundTrippableTypes = sets.NewString("List", "ListOptions")
+var nonInternalRoundTrippableTypes = sets.NewString("List", "ListOptions", "WatchEvent")
```

Comment:
WatchEvent doesn't have TypeMeta.

On Thu, Jul 7, 2016 at 9:49 AM, Michail Kargakis <notifications@github.com>
wrote:

> In pkg/api/serialization_test.go
> <https://github.com/openshift/origin/pull/9578#discussion_r69909080>:
>
> >  		fuzzInternalObject(t, v1.SchemeGroupVersion, item, seed)
> >  		roundTrip(t, kapi.Codecs.LegacyCodec(v1.SchemeGroupVersion), item)
> >  	}
> >  }
> >
> >  // Keep this in sync with the respective upstream set
> > -var nonInternalRoundTrippableTypes = sets.NewString("List", "ListOptions")
> > +var nonInternalRoundTrippableTypes = sets.NewString("List", "ListOptions", "WatchEvent")
>
> Can you link to a description of why this is not roundtrippable? We
> already have an issue for eliminating this list: #5529
> <https://github.com/openshift/origin/issues/5529>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9578/files/6031f09354bce5f9446da32892b34fe285590483..ae5104aac39f6c8bdbec8258ed7b427aa4a24d18#r69909080>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p3K4eDLCTpiAvBm7YEDobB0FkZcdks5qTQPYgaJpZM4I-xTE>
> .
>